modules/aggregation-layers/src/utils/gpu-grid-aggregation/aggregate-all-vs-64.glsl.js

```diff
@@ -36,7 +37,7 @@ void main(void) {
 
   gl_Position = vec4(pos, 0.0, 1.0);
 
-  float yIndex = floor(float(gl_InstanceID) / gridSize[0]);
+  float yIndex = floor((float(gl_InstanceID) / gridSize[0]) + ROUNDING_ERROR);
```

Comment:
Have you tried making this an integer division. Then you wouldn't even need the `floor`

```
uniform uvec2 gridSize;
...
float yIndex = float(gl_InstanceID / gridSize[0]);
```

Reply:
fixed.